index.js

```diff
@@ -37,50 +123,92 @@ module.exports = {
         'vite',
         '@rollup/plugin-babel',
         'decorator-transforms',
+
+        // Dependencies out of date from upstream
+        // (or waiting for the release train to catch up)
+        '@ember/string',
+        '@ember/test-helpers',
+        'ember-resolver',
+        'eslint-plugin-ember',
+        'ember-template-lint',
+        'qunit-dom',
+        'qunit',
+        'stylelint',
+        'stylelint-prettier',
+        'stylelint-config-standard',
+        'concurrently',
+        'prettier-plugin-ember-template-tag',
+        'ember-route-template',
+
+        // TypeScript
+        // Note that Vite supports TypeScript with 0 configuration on the user's part
+        ...(options.typescript
+          ? [
+              '@babel/plugin-transform-typescript',
+              'typescript',
+              '@glint/core@unstable',
+              '@glint/environment-ember-loose@unstable', // currently required :(
+              '@glint/environment-ember-template-imports@unstable',
+              '@glint/template@unstable',
+              '@typescript-eslint/eslint-plugin',
+              '@typescript-eslint/parser',
+            ]
+          : []),
       ],
       packageManager: options.packageManager,
     });
+  },
 
-    let uninstallTask = this.taskFor('npm-uninstall');
-    const packages = [
-      'ember-fetch',
-      'broccoli-asset-rev',
-      'ember-cli-app-version',
-      'ember-cli-clean-css',
-      'ember-cli-dependency-checker',
-      'ember-cli-sri',
-      'ember-cli-terser',
+  async afterInstall(options) {
+    const filesToDelete = [
+      // now in the project root
+      'app/index.html',
+      // replaced with application.gjs/gts
+      'app/templates/application.hbs',
+      // replaced with .eslintrc.cjs
```

Comment:
shouldnt' this be something that we do upstream in ember-cli? if we want it to be a `.cjs` file?